packages/next/src/client/components/app-router-headers.ts

```diff
@@ -5,8 +5,6 @@ export const NEXT_ROUTER_STATE_TREE = 'Next-Router-State-Tree' as const
 export const NEXT_ROUTER_PREFETCH_HEADER = 'Next-Router-Prefetch' as const
 export const NEXT_URL = 'Next-Url' as const
 export const RSC_CONTENT_TYPE_HEADER = 'text/x-component' as const
-export const RSC_VARY_HEADER =
```

Comment:
This is removed as a constant to avoid any assumptions about the `Vary` header being static. Instead we inline the vary header in places where we're setting it.